indico/modules/auth/templates/login_page.html

```diff
@@ -57,20 +65,19 @@
             {% endif %}
             <div class="login-providers external">
                 {% for provider in external_providers %}
+                    {% set provider_logo = provider.settings.get('logo_url') %}
                     <a href="{{ url_for('.login', provider=provider.name, _method='GET') }}"
-                       class="i-button external-provider-{{ provider.name }}">
+                       class="i-button external-provider-{{ provider.name }}"
+                       {% if provider_logo %}
+                           style="background-image: url('{{ provider_logo }}'); background-repeat: no-repeat;"
```

Comment:
What do you think about adding `background-size: contain;` as well? This would make things much nicer in case the logo image is a bit bigger than the button.

Reply:
I added it to he PR for now, please let me know if this works well for the logos you're using (I just tried it with the slightly bigger CERN logo). Once you confirmed that I'll merge the PR.

Reply:
Sure.

Reply:
TESTED: it works WELL, thanks.